nixos/modules/services/hardware/amdgpu.nix

```diff
@@ -0,0 +1,42 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.amdgpu;
+in {
+  options.hardware.amdgpu = {
+    legacySupport.enable = lib.mkEnableOption ''support for old AMD graphics cards,
+      this uses `amdgpu` instead of `radeonsi` driver for Southern Islands (Radeon HD 7000)
+      series and Sea Islands (Radeon HD 7000) series cards.
+      Note: this does NOT add support for analog video output graphics cards.
```

Comment:
It does not _not add_ support, it _removes_ support. Analog output works on GPUs that have it with radeonsi, but not with amdgpu.